src/extension/__tests__/actor.test.ts

```diff
@@ -29,171 +29,154 @@ function createTestAdapter<Messages = unknown>() {
 }
 
 test("sends messages to specified adapter in devtools message format", () => {
-  type Message = { type: "test"; payload: string };
```

Comment:
The one downside to this change is that we lose the ability to do generic messages in tests and now must use the real message types. This may be a bit more brittle over time as message types are refactored, but I think its a fair tradeoff.